src/garage/_dtypes.py

```diff
@@ -77,6 +78,8 @@ class EpisodeBatch(
         lengths (numpy.ndarray): An integer numpy array of shape :math:`(N,)`
             containing the length of each episode in this batch. This may be
             used to reconstruct the individual episodes.
+        episode_infos (dict): A dict of numpy arrays containing the episode-level
```

Comment:
i think you should list this as the first member/argument, since logically its existence precedes everything else